frontend/context/app.tsx

```diff
@@ -105,6 +112,7 @@ type InitialStateType = {
   sandboxExpiry?: string;
   noSandboxHosts?: boolean;
   filteredHostsPath?: string;
+  filteredSoftwarePath?: string;
```

Comment:
not sure this is relevant but is this to go back to the previous URL? If so maybe it would be easier to use the router `goBack` method when the button in the UI code?